openedx/features/marketplace/views.py

```diff
@@ -12,11 +16,16 @@ def get(self, request, *args, **kwargs):
         return render_to_response(self.template_name, {})
 
 
-class MarketplaceCreateRequestView(View):
+@method_decorator(login_required, name='dispatch')
```

Comment:
Can we use `https://docs.djangoproject.com/en/1.11/topics/auth/default/#django.contrib.auth.mixins.LoginRequiredMixin`